.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ 2.5, 2.6, 2.7, 3.0 ]
+        ruby: [ 2.5, 2.6, 2.7, '3.0', 3.1 ]
```

Comment:
Do you mind quoting all of them for uniformity? 🙏 

That will avoid us forbidding the quotes again if we reach Ruby 4.0 someday.